app/models/manageiq/providers/nuage/network_manager/floating_ip.rb

```diff
@@ -1,2 +1,40 @@
 class ManageIQ::Providers::Nuage::NetworkManager::FloatingIp < ::FloatingIp
+  supports :delete
+
+  def delete_floating_ip_queue(userid)
```

Comment:
we don;t have to have explicit todo here, but we should move this to core too. Only the ```raw_``` method and ```supports <action>```, should be in provider. (for shared actions)

But lets do this in follow up PR